README.md

```diff
@@ -54,19 +54,19 @@ you must add the following dependencies:
 composer require symfony/dependency-injection symfony/http-kernel
 ```
 
-Enable the bundle on the `AppKernel` class:
+Register the bundle in the kernel of your application:
 
 ``` php
 // app/AppKernel.php
 
 public function registerBundles()
 {
     $bundles = array(
-        // ...
+        // …
```

Comment:
Wow not sure about that at all!

I don't have a `…` key on my keyboard. :stuck_out_tongue: 

Is that really necessary?